src/wmnds/components/content-tiles/travel-updates/travel-updates.njk

```diff
@@ -1,19 +1,182 @@
 {% macro wmndsContentTileTravelUpdates(params) %}
-<div class="wmnds-content-tile wmnds-content-tile-container">
-      <h2 class="wmnds-content-tile__title">Travel updates</h2>
+<div class="wmnds-content-tile wmnds-content-tile--container">
+   <h2 class="wmnds-content-tile__title">Travel updates</h2>
    <div class="wmnds-content-tile__copy">
       <p>All routes through this disruption.</p>
    </div>
-<h3>Roads</h3>
-   {%- from "wmnds/components/disruption-indicator/large/_large.njk" import wmndsDisruptionIndicatorLarge as primary -%}
+   <h3>Roads</h3>
+   {%- from "wmnds/components/disruption-indicator/large/_large.njk" import wmndsDisruptionIndicatorLarge as large -%}
+   {%- from "wmnds/components/disruption-indicator/medium/_medium.njk" import wmndsDisruptionIndicatorMedium as medium -%}
+   {%- from "wmnds/components/disruption-indicator/medium/_medium.njk" import wmndsDisruptionIndicatorMedium as star -%}
+   <div class="wmnds-content-tile--container__wrapper">
       {{
-        primary({
-          strongText: "Hagley Road",
-          text: 'Usual traffic',
-          iconLeft: "modes-isolated-roads",
-          iconRight: 'general-success'
-        })
+      large({
+      strongText: "Hagley Road",
+      text: 'Usual traffic',
+      iconLeft: "modes-isolated-roads",
+      iconRight: 'general-success'
+      })
       }}
+   </div>
+   <div class="wmnds-content-tile--container__wrapper">
+      {{
+      large({
+      strongText: "Ring Road",
+      text: 'Usual traffic',
+      iconLeft: "modes-isolated-roads",
+      iconRight: 'general-success'
+      })
+      }}
+   </div>
+   <h3>Bus Routes</h3>
+   <div class="wmnds-col-1-3 wmnds-col-md-auto">
+         <div class="wmnds-content-tile--container__bus">
```

Comment:
Again this is a weird BEM class issue as mentioned above...fix these throughout the file.